### DISCUSSION TITLE
debugger taking time to start

### DISCUSSION THREAD

**Discussion by rkadam61001** [microsoft/vscode-discussions#1153] [Extension Development QnA]
Can anyone help me with this issue "&"warning: GDB: Failed to set controlling terminal: Operation not permitted\n"".
Whenever I do "ctrl+shift+P" and then do debug i get the message which i mentioned above.And it take around 10 min to start debug

**Comment by gjsjohnmurray** [comment]
Are you **developing** an extension? If not, please see the [READ FIRST](https://github.com/microsoft/vscode-discussions/discussions/5) article which is pinned to the top of this discussion forum and contains pointers to more appropriate places for bug reports, questions etc.

**Comment by burkeholland** [comment]
This appears to be off topic for extension development. I'm closing for now. If you feel this is an error, please feel free to reopen.
